pkg/client/cache/listers.go

```diff
@@ -106,6 +107,44 @@ func (s *StoreToNodeLister) List() (machines api.NodeList, err error) {
 	return machines, nil
 }
 
+func (s *StoreToNodeLister) NodeCondition(conditionType api.NodeConditionType, conditionStatus api.ConditionStatus) storeToNodeConditionLister {
```

Comment:
Yeah still a TODO; I don't think our selector is advanced enough to specify the condition(s) we care about at the moment. But eventually we'd want to do this server side. Presumably it'd be maximally efficient if we push doing this work down into our storage layer.